srcpkgs/when/template

```diff
@@ -1,15 +1,19 @@
 # Template file for 'when'
 pkgname=when
-version=1.1.40
+version=1.1.41
 revision=1
-wrksrc=when_dist
+hostmakedepends="perl"
 depends="perl"
 short_desc="Minimalistic personal calendar program"
 maintainer="Leah Neukirchen <leah@vuxu.org>"
 license="GPL-2.0-only"
 homepage="http://www.lightandmatter.com/when/when.html"
-distfiles="http://www.lightandmatter.com/when/when.tar.gz>when-${version}.tar.gz"
-checksum=2224cdba55cd4b3776cc328fbec2758aae4ccab3bf8acfc03ecab66591aedeb3
+distfiles="https://github.com/bcrowell/when/archive/refs/tags/${version}.tar.gz>when-${version}.tar.gz"
+checksum=78bfd3f18e0d3e42797c7aacb16829d2354d935f12e04db9e467d4bec389c884
+
+do_build() {
+	make when.1
+}
 
 do_install() {
 	vbin when
```

Comment:
`make install` can replace do_build and do_install
```suggestion
build_style=gnu-makefile
```

Reply:
The makefile tries to infer the location of the perl executable and this results in a non working script and messes up with the man file as well. Should I leave things as is or use the build_style and edit the makefile ?

Reply:
Let's leave it like this then, but I think it would be nice to have a proper makefile upstream.